SmartDeviceLink/SDLSystemCapabilityManager.h

```diff
@@ -198,6 +212,37 @@ typedef void (^SDLUpdateCapabilityHandler)(NSError * _Nullable error, SDLSystemC
  */
 - (void)updateCapabilityType:(SDLSystemCapabilityType)type completionHandler:(SDLUpdateCapabilityHandler)handler;
 
+/**
+ Subscribe to a particular capability type using a block callback
+
+ @param type The type of capability to subscribe to
+ @param block The block to be called when the capability is updated
+ @return An object that can be used to unsubscribe the block using unsubscribeFromCapabilityType:withObserver: by passing it in the observer callback, or nil if subscriptions aren't available on this head unit
+ */
+- (nullable id<NSObject>)subscribeToCapabilityType:(SDLSystemCapabilityType)type usingBlock:(SDLCapabilityUpdateHandler)block;
```

Comment:
When I subscribe to system capability type `APP_SERVICES` using the block callback, I get updated when a `SDLOnSystemCapabilityUpdated` notification is received except when the `updateReason` is `REMOVED`.

Reply:
I cannot see why. When I built a similar unit test path it all worked as expected and looking through the code, there's no different code path.

Reply:
This was my bad. I was expecting to get the `OnSystemCapabilityUpdated` notification as the system capability but instead it is `nil` because the `SDLSystemCapabilityManager` handles removing the capability from the the saved capabilities/